python/Ganga/test/GPI/Tasks/TestCoreTasks.py

```diff
@@ -6,24 +6,26 @@
 class TestCoreTasks(GangaUnitTest):
     """A set of tests to cover both ITask and CoreTask plus related objects"""
 
+    argList = ['300', '300', '300']
+
     def setUp(self):
         """Make sure that the Tasks object isn't destroyed between tests"""
         super(TestCoreTasks, self).setUp()
         from Ganga.Utility.Config import setConfigOption
         setConfigOption('TestingFramework', 'AutoCleanup', 'False')
         setConfigOption('Tasks', 'TaskLoopFrequency', 1)
-        self._numTasks = 50
+        self._numTasks = 25
 
     def createTask(self):
         """create a task with some defaults"""
         from Ganga.GPI import tasks, CoreTask, CoreTransform, Executable, GenericSplitter
         t = CoreTask()
 
         trf = CoreTransform()
-        trf.application = Executable()
+        trf.application = Executable(exe='sleep')
```

Comment:
Fair enough but maybe reduce the sleep time a bit? It would good to have these completing while the Task is submitting, etc.